tests/async/test_appeal_file_view.py

```diff
@@ -268,3 +268,124 @@ def test_appeal_file_view_authenticated_incorrect(self):
         response = self.client.get(
             reverse("appeal_file_view", kwargs={"appeal_uuid": self.appeal.uuid})
         )
+
+    def test_appeal_file_view_missing_file(self):
+        # Test when the file is missing but appeal exists
+        self.appeal.document_enc = None
+        self.appeal.save()
```

Comment:
Modifying the global is not great right? Maybe make a new appeal without document.